components/kubeconfig-service/pkg/runtime/runtime.go

```diff
@@ -48,8 +48,9 @@ var L2L3OperatorPolicyRule = map[string][]rbacv1.PolicyRule{
 		rbacv1helpers.NewRule("*").URLs("*").RuleOrDie(),
 	},
 	RUNTIME_OPERATOR: {
-		rbacv1helpers.NewRule("get", "list", "watch").Groups("*").Resources("*").RuleOrDie(),
+		rbacv1helpers.NewRule("*").Groups("*").Resources("pods/ephemeralcontainers").RuleOrDie(),
 		rbacv1helpers.NewRule("get", "list", "watch").URLs("*").RuleOrDie(),
+		rbacv1helpers.NewRule("*").URLs("*").RuleOrDie(),
```

Comment:
all permissions for non-resource URLs for L2 operators?? Why, to create a backdoor exploitable by L2 ops?

```suggestion
```